tunnel/ssh_keys.go

```diff
@@ -0,0 +1,54 @@
+package tunnel
+
+import (
+	"crypto/x509"
+	"encoding/pem"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"syscall"
+
+	log "github.com/sirupsen/logrus"
+	"golang.org/x/crypto/ssh"
+	"golang.org/x/crypto/ssh/terminal"
+)
+
+func privateKeyFile(path string) (ssh.AuthMethod, error) {
```

Comment:
I think this function could use a better name. Maybe something like `readPrivateKeyFile`